lib/connect.js

```diff
@@ -1251,23 +1268,23 @@ regular).
         return data;
     }
 
-    function _get(route, params, responseType, responseTransformer) {
-        return request(route, "GET", params || {}, responseType, responseTransformer);
+    function _get(route, params, responseType, responseTransformer, postJson = false) {
```

Comment:
change flag `postJson` to something generic like `isJSON` or just `json`.

Reply:
Done. Changed it to **isJSON**.